products/yarn.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Yarn
+category: server-app
+tags: javascript
+iconSlug: yarn
+permalink: /yarn
+versionCommand: yarn --version
+changelogTemplate: https://github.com/yarnpkg/berry/releases/tag/%40yarnpkg%2Fcli%2F__LATEST__
+activeSupportColumn: false
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:github/yarnpkg/berry
+-   purl: pkg:github/yarnpkg/yarn
+-   purl: pkg:npm/yarn
+-   purl: pkg:npm/yarnpkg/cli
+-   repology: yarn
+
+auto:
+-   npm: yarn
+-   npm: "@yarnpkg/cli"
+
+# 4 is still in RC
+releases:
+-   releaseCycle: "3"
+    releaseDate: 2021-07-26
+    eol: false
+    latestReleaseDate: 2023-06-30
+    latest: '3.6.1'
+
+-   releaseCycle: "2"
+    releaseDate: 2020-07-09
+    eol: 2021-09-06
+    latestReleaseDate: 2021-09-06
+    latest: '2.4.3'
+
+-   releaseCycle: "1"
+    releaseDate: 2017-09-05
+    eol: false
+    latestReleaseDate: 2022-06-08
+    latest: '1.22.19'
+
+---
+
+> [Yarn](https://yarnpkg.com/) is a modern package manager for the Javascript ecosystem. It offers
+  extensibility via plugins, and a Plug'n'Play strategy that supports various frameworks and tools.
+
+Yarn Classic (v1) entered maintenance mode in January 2020 and will eventually reach
```

Comment:
Adding a link to the source of this information (https://github.com/yarnpkg/yarn/issues/8583 or https://dev.to/arcanis/introducing-yarn-2-4eh1) and what it means (only vulnerabilities will be fixed) would be great.